.github/workflows/pull-request.yml

```diff
@@ -13,7 +13,7 @@ env:
 jobs:
   setup:
     if: github.event.pull_request.draft == false
-    runs-on: ubuntu-latest
+    runs-on: ubuntu-20.04
```

Comment:
Ubutntu latest (24.04) has an issue with some packages, so the github actions were never installing. I downgraded for the time being.

Reply:
Good catch!